doc/contributing.rst

```diff
@@ -398,6 +398,26 @@ Code reviews
    saves time explaining the change or applying it; see also
    https://forum.nim-lang.org/t/4317
 
+2. When reviewing large diffs that may involve code moving around, github's interface
+   doesn't help much as it doesn't highlight moves. Instead you can use something
+   like this, see visual results [here](https://github.com/nim-lang/Nim/pull/10431#issuecomment-456968196):
```

Comment:
This needs to be rst-style link:

```
`here <https://github.com/nim-lang/Nim/pull/10431#issuecomment-456968196>`_
```

Reply:
didn't @araq add markdown interoperability in rst processing?

Reply:
This file will be viewed on Github ;)

Reply:
sigh, was really hoping to be able to use markdown features :); ok done